synapse/tests/test_axon.py

```diff
@@ -507,15 +507,17 @@ def test_axon_host_spinbackup(self):
                 busurl = 'tcp://127.0.0.1:{}/axonbus'.format(port)
                 hstcfg['vars']['hcfg0']['axon:axonbus'] = busurl
                 hstcfg['vars']['hcfg1']['axon:axonbus'] = busurl
+                svcbus = s_service.openurl('tcp://127.0.0.1:0/axonbus', port=port)  # type: s_service.SvcProxy
 
                 with s_daemon.Daemon() as axondmon:
+                    w = svcbus.waiter(6, 'syn:svc:init')
                     axondmon.loadDmonConf(hstcfg)
-                    svcbus = s_service.openurl('tcp://127.0.0.1:0/axonbus', port=port)  # type: s_service.SvcProxy
-                    time.sleep(2)
+                    w.wait(30)
```

Comment:
Tests still fail for me w/ 2s wait. I bumped it to 30s and can't get it to fail locally. This will obviously differ based on the computer the tests are being run on. Increasing the wait time shouldn't hurt anything, as faster test runners will stop waiting once all of the init events are set.